inc/namespace.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Altis\Chassis;
+
+/**
+ * Bootstrap.
+ */
+function bootstrap() {
+	add_filter( 'qm/output/file_path_map', __NAMESPACE__ . '\\set_file_path_map', 1 );
+}
+
+/**
+ * Enables Query Monitor to map paths to their
+ * original values on the host.
+ *
+ * @param array $map an associative array of folders as keys, and their mappings as values
```

Comment:
Can simplify this (and correct grammar):

```suggestion
 * @param array $map Map of guest path => host path
```